caiman/source_extraction/volpy/mrcnn/config.py

```diff
@@ -0,0 +1,236 @@
+"""
+Mask R-CNN
+Base Configurations class.
+
+Copyright (c) 2017 Matterport, Inc.
+Licensed under the MIT License (see LICENSE for details)
```

Comment:
References a license file that's not included. This needs to be fixed.

Reply:
So Pat, what should we do here?